index.js

```diff
@@ -20,39 +20,38 @@
 //. purpose: the [`typeof`][1] operator and [`Object.prototype.toString`][2].
 //. Each has pros and cons, but neither supports user-defined types.
 //.
-//. This package specifies an [algorithm][3] for deriving a _type identifier_
-//. from any JavaScript value, and exports an implementation of the algorithm.
-//. Authors of algebraic data types may follow this specification in order to
-//. make their data types compatible with the algorithm.
+//. This package exports a function which derives a _type identifier_ from any
+//. JavaScript value, and a [specification][3] for customizing the type
+//. identifier of a value.
```

Comment:
This sentence could be improved. As written _This package exports_ applies to both clauses, but the package does not export a specification.

I suggest:

> sanctuary-type-identifiers comprises:
>
>   - an npm and browser -compatible package for deriving the _type identifier_ of a JavaScript value; and
>   - a specification which authors may follow to specify type identifiers for their types.